### DISCUSSION TITLE
primal/dual residuals scaled or not ?

### DISCUSSION THREAD

**Discussion by charleskhazoom** [osqp/osqp#612] [General]
Hi,

I would like to know whether the primal and dual residual values returned by OSQP are scaled or not. I am guessing that the termination criterias and returned residual values are unscaled by default (unless `scaled_termination=1`). 

Can someone confirm this ?

**Comment by goulart-paul** [comment]
I believe that this is correct.   The relevant line is here: https://github.com/osqp/osqp/blob/17048703bc5baeca0c5f8994de15cb411c82e4a1/src/auxil.c#L269
